src/main/webapp/js/common.js

```diff
@@ -772,7 +775,7 @@ function isInstitutionValid(rawInstitution) {
         return false;
     }
     
-    if (institution.match(/[^\/\\,.'\-\(\)0-9a-zA-Z \t]/)) {
+    if (institution.match(/[^/\\,.'\-()0-9a-zA-Z \t]/)) {
```

Comment:
Is this regex duplicated in line 753? In that case we can define it once in some appropriate place (may be as a `FirstTimer` issue)